organising/organising-a-meeting.md

```diff
@@ -0,0 +1,164 @@
+
+* set ticket sales to close at 5:30pm on the day
+* limit total ticket sales to 95
+* add a link to the zoom meeting as a plain link to the "online event page" - we haven't linked our zoom account, and the link works just fine.
+* check that there are scheduled emails to attendees on the day to send the link out to them before 6:30pm
+* enable the waiting list in the off chance the meeting "sells out"
+
+### Schedule the zoom meeting
+
+We schedule the zoom meeting in advance so we can:
+
+1. add the link to the "online event page" in eventbrite
+2. pass on the link to the speakers well in advance
+3. get the recording properly named for the meeting
+
+### Write up on lrug.org
+
+The write-ups are fairly straightforward, so much so you can copy the previous meeting page and replace the details.
+
+1. Change the registration url to the new eventbrite page
+2. Fill in the details of each talk in the agenda section
+3. Update the frontmatter to include `created_at` / `published_at` timestamps that are _broadly_ correct (they're used to order the pages)
+4. Make sure the date and times are correct for the announcement
+5. If you feel up to it tweak some of the wording to make the announcement unique, but this is far from a requirement - after all there are only so many way to say "please register", "hang out afterwards", "here are the talks"
+
+Push this up as a PR to [the lrug.org github repo][github-lrug.org] and once CI has okayed it you can merge and this will automatically deploy to lrug.org.
+
+### Tell the mailing list about the meeting
+
+Once the meeting is written up on lrug.org you should email [the mailing list][mailing-list] to tell people about it.  We keep this short and to the point:
+
+1. Include the date and start and end times
+2. Include titles and speaker names
+3. Point people at lrug.org to find out more
+4. Point people at the readme to agree to the code of conduct
+5. Point people at eventbrite to register for a place
+6. Ask for more volunteers - even if the agenda is full, it can be a good hook for future volunteers
+
+### Tweet about the meeting
+
+Once the meeting is written up on lrug.org you should tweet about it.  Twitter has even less space than email, so keep it brief and point people at lrug.org for full details.  Feel free to use threads, but no-one wants a 30-tweet storm to announce a meeting.
+
+1. Mention the speakers by their twitter handles if you have them
+2. Point people at lrug.org for full details and registration info
+
+It can be useful to tweet a few times in the lead up, and to tweet on the morning of the actual meeting.  A lot of people need reminding and twitter is as good a way as any.
+
+## Host the meeting
+
+This is quite involved, so [check out our separate "hosting the meeting" documentation](/organising/hosting-a-meeting) for all the detail.  The short version is that you need to:
```

Comment:
```suggestion
This is quite involved, so [check out our separate "hosting the meeting" documentation](/organising/hosting-a-meeting) for all the detail.

It's also important to remember that just because you're responsible for organising the meeting, it doesn't mean that you need to be the person hosting the meeting that month. Yours is the power to delegate!

The short version is that you need to:
```